apps/frontend/components/Filters/DisasterFilter.tsx

```diff
@@ -1,81 +1,86 @@
 import {
-  Card,
-  CardContent,
-  FormControlLabel,
-  Radio,
-  RadioGroup,
-  Typography,
+  Box,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
 } from '@mui/material';
-import { DisasterMapping, IncidentMapping } from '@wfp-dmp/interfaces';
-import { ChangeEvent } from 'react';
+import { DisasterMapping, INCIDENT } from '@wfp-dmp/interfaces';
+import { useState } from 'react';
 import { FormattedMessage, useIntl } from 'react-intl';
 
+import { IncidentSelect } from './IncidentSelect';
+
+const floodDroughtKeys = Object.keys(DisasterMapping).map(
+  disaster => DisasterMapping[disaster],
+);
+const disasterKeys = [...floodDroughtKeys, INCIDENT];
+
 interface Props {
-  value: string;
-  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  value: string[];
+  onChange: (newValue: string[]) => void;
 }
 
 export const DisasterFilter = ({ value, onChange }: Props): JSX.Element => {
   const intl = useIntl();
-
-  const disasters = Object.keys(DisasterMapping).map(
-    disaster => DisasterMapping[disaster],
+  // value can contain more than one element only for incident
+  //disaster type value "1", "2" or INCIDENT
+  const [disasterType, setDisasterType] = useState(
```

Comment:
I think the logic could be a bit cleaner here.

DisasterType could be `FLOOD | DROUGHT | INCIDENT` for example, and then use inicidentType to set the value `1-10`?